File: BaoLinngQiu/Assets/Scripts/ballzizhuan1.cs
using UnityEngine;
using System.Collections;

public class ballzizhuan1 : MonoBehaviour {
	private bool flag1 =false;
	private string wenlilili;
	public GameObject guangyun;
	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
		if(Input.GetMouseButtonDown(0)) {
			Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
			RaycastHit hit;
			if(Physics.Raycast(ray, out hit))
			{
				if(hit.transform.root.transform==this.transform)	      
				{	
					flag1=true;
					guangyun.transform.position = this.transform.position;

					wenlilili = this.renderer.material.mainTexture.name;
					PlayerPrefs.SetString("wl",wenlilili);		
					Debug.Log(wenlilili);
				}
				else 
					flag1 = false;

			}      	
		}
		if(flag1)
		{
			this.transform.Rotate(-Time.deltaTime * 500,0,0);
		}else{
			this.transform.Rotate(-Time.deltaTime * 50,0,0);
		}
	}
}

File: BaoLinngQiu/Assets/Script001/zizhuan/ballzizhuan1.js
static var flag1 : boolean=true;

function Update () 
{	
	if(Input.GetMouseButtonDown(0)) {
   		var ray = Camera.main.ScreenPointToRay(Input.mousePosition);
        var hit : RaycastHit;
		if (Physics.Raycast (ray, hit))
	    {
            if(hit.transform.root.transform==this.transform)	      
            {	
            	flag1=true;
            	ballzizhuan2.flag2=false;
            	ballzizhuan3.flag3=false;
            	ballzizhuan4.flag4=false;
            	ballzizhuan5.flag5=false;
            	ballzizhuan6.flag6=false;
            	PlayerPrefs.SetString("wl","wenli1");

            }
	    }      	
	}
	if(flag1)
	{
 		this.transform.Rotate(-Time.deltaTime * 500,0,0);
	}else{
		this.transform.Rotate(-Time.deltaTime * 50,0,0);
	}
}